tiledb/CMakeLists.txt

```diff
@@ -927,7 +927,16 @@ endif()
 # TileDB static and shared library targets
 ############################################################
 
-add_library(tiledb_shared SHARED $<TARGET_OBJECTS:TILEDB_CORE_OBJECTS>)
+if(WIN32)
+  # The .rc file will be created elsewhere (currently below) via configure_file()
+  add_library(tiledb_shared SHARED $<TARGET_OBJECTS:TILEDB_CORE_OBJECTS> ${CMAKE_CURRENT_BINARY_DIR}/tiledb_win32_verinfo.rc)
```

Comment:
This can be written as `target_sources(tiledb_shared ...)` after `add_library` so that there's only one call to `add_library`.

Reply:
changed